client/src/components/LandingFooter/index.tsx

```diff
@@ -0,0 +1,46 @@
+// @ts-nocheck
```

Comment:
same with all the LandingFooter files, this was moved in the `landing` repo a while back